modules/fontrendering/include/modules/fontrendering/textrenderer.h

```diff
@@ -109,7 +109,7 @@ struct TextTextureObject {
  */
 class IVW_MODULE_FONTRENDERING_API TextRenderer {
 public:
-    TextRenderer(const std::string& fontPath = util::getDefaultFontPath() + "/arial.ttf");
+    TextRenderer(const std::string &fontPath = util::getDefaultFontPath() + "/OpenSans-semibold.ttf");
```

Comment:
maybe we should have a util::getDefaultFont()? could even be backed by a setting a some point...

Reply:
good idea. Will add one.